soh/soh/SohMenuBar.cpp

```diff
@@ -508,6 +508,7 @@ void DrawEnhancementsMenu() {
                 UIWidgets::Tooltip("Allows you to change the number of days it takes for Biggoron to forge the Biggoron Sword");
                 UIWidgets::PaddedEnhancementSliderInt("Vine/Ladder Climb speed +%d", "##CLIMBSPEED", "gClimbSpeed", 0, 12, "", 0, true, false, true);
                 UIWidgets::PaddedEnhancementSliderInt("Block pushing speed +%d", "##BLOCKSPEED", "gFasterBlockPush", 0, 5, "", 0, true, false, true);
+                UIWidgets::PaddedEnhancementSliderInt("Crawl speed %dx", "##CRAWLSPEED", "gCrawlSpeed", 1, 5, "", 0, true, false, true);
```

Comment:
The 0 here needs to be changed to a 1.  Since you're only using values 1 through 5, the default value should be 1, since that will trigger the default value in the logic, not 0, which is currently unhandled.

Reply:
Done! Lot of remnants from initially intending to unify with Climb Speed, whoops.